### ISSUE TITLE
Unable to Redo deletion of Paragraph blocks with non-breaking space in content

### ISSUE THREAD

**Issue by guarani** [WordPress/gutenberg#27656]
**Describe the bug**

I'm unable to redo the action of deleting Paragraph blocks when they contain non-breaking spaces (i.e. `&nbsp;`).

**To Reproduce**
1. Open the mobile editor
2. Add two Paragraph blocks, each containing the text "Hello world"
3. Switch to HTML mode and replace the space in the first block with `&nbsp;`, return to the Visual editor
4. Delete the two blocks
5. Tap the Undo button until both block re-appear
6. Tap the Redo button to delete the two blocks again
7. Notice that the Redo button is disabled

**Expected behavior**

**Screenshots**

| | |
| - | - |
| <img src="https://user-images.githubusercontent.com/1898325/101823889-3469e900-3b0a-11eb-81de-00e272a86f03.gif" alt="" width="350"> |

**Smartphone (please complete the following information):**
 - Device: iPhone 11
 - OS: iOS 14.2
 - Version WPiOS 16.2 (Gutenberg Mobile 1.41)

**Comment by kathrynwp** [comment]
@dcalhoun Hi there! Would you be able to take a peek and see if this is still an issue, or if it can be closed? Thanks!

**Comment by dcalhoun** [comment]
@kathrynwp yes, I am able to reproduce this issue on an iPhone 13 mini with the WordPress 20.5 app.

**Comment by tyxla** [comment]
This is no longer relevant after the React Native implementation was removed in #78747. Closing.
